src/common/transformations/include/transformations/common_optimizations/transpose_sinking_binary.hpp

```diff
@@ -11,20 +11,20 @@
 namespace ov {
 namespace pass {
 
-class TRANSFORMATIONS_API TransposeSinkingBinaryElementwiseForward;
-class TRANSFORMATIONS_API TransposeSinkingBinaryElementwiseBackward;
+class TRANSFORMATIONS_API TransposeSinkingBinaryForward;
+class TRANSFORMATIONS_API TransposeSinkingBinaryBackward;
```

Comment:
why is it remaned to `TransposeSinkingBinaryForward`? The `BinaryForward` sounds confusing.

Reply:
I renamed it from BinaryElementwise to Binary since this transformation proceeded only BinaryEltwise but now it proceeds also PRelu that is not binary elementwise operation. The other way is just copy/paste entire BinaryElementwise forward and backward transformations and call them differently apart + copy entire unit tests for the new transformations. I choose the first way